### ISSUE TITLE
Persist more audio player settings

### ISSUE THREAD

**Issue by imthath-m** [quran/quran-ios#678]
Currently, AFAIK, only the choice of the reciter is persisted from this screen.

Can we also persist the choices in the screen for the options "Play each verse" and "Play set of verses"? Just saving them in user defaults could do the job for now, so that whenever user plays, it is the same choice from the previous time they played.

![IMG_5954](https://github.com/user-attachments/assets/a3757645-d145-4414-bf8f-d80275de2b36)

Every single time, I find myself modifying it to play each verse 3 times and the entire range 3 times. Would be better if it is persisted.

**Comment by mohamede1945** [comment]
Agreed. There’s also the common use case of users switching from memorization to listening and wanting to switch back. I think we should provide an easy way to reset this when playing audio that relies on persisted values.
